original/v1/s001-install/instalace.html

```diff
@@ -63,10 +63,29 @@ <h3 class="text-muted">PyLadies Brno</h3>
 
                     <h3>macOS</h3>
                     <p>
-                        Drž se instrukcí pro Windows, ale stáhni instalátor pro <em>Mac OS X</em>,
-                        podle verze tvého systému.
+                        Nainstalujeme si nástroj homebrew. <a href="http://brew.sh">Homebrew</a> za nás řeší a zjednodušuje instalaci aplikací a knihoven, které budeme potřebovat pro programování.
                     </p>
 
+                    <p>
+                        Spusť v příkazové řádce příkaz:
+                    </p>
+
+<pre>
+$ <b>/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"</b>
+</pre>
+
+                    <div class="note">
+                        Instalace gitu při instalaci homebrew by si měla sama vyžádat command line developer tools. Pokud se tak nestane stačí zadat xcode-select --install a nainstalovat je ručně.
```

Comment:
Mně zas Homebrew přežil už několikeré povýšení včetně toho posledního na Sierru. Dokonce i přechod na El Capitana to zvládlo, kdy se zaváděla ochrana systémových souborů. Jen teď nedávno se měnilo výchozí nastavení, kam se budou balíčky instalovat, ani s tím jsem ale nenarazil na žádnou nečekanou složitost – stačilo udělat jeden krok podle návodu.